src/app/workflow/dag/state/dag.service.ts

```diff
@@ -285,11 +285,12 @@ export class DagService {
    */
   private createPopupHTML(name: string, runText: string, cyElement: HTMLDivElement): HTMLDivElement {
     const div = document.createElement('div');
+    const dagTooltipAttribute = 'dag-tooltip';
     div.innerHTML = `
     <div class="qtip-titlebar">${name}</div>
     <div class="qtip-content">${runText}</div>
     `;
-    div.setAttribute('class', 'opaq qtip-bootstrap bootstrap-tooltip-z-index');
+    div.setAttribute('class', 'opaq qtip-bootstrap bootstrap-tooltip-z-index' + dagTooltipAttribute);
```

Comment:
There's no space between `'... bootstrap-tooltip-z-index'` and `dagTooltipAttribute` (I think you had a space before)